src/openrct2/ride/Vehicle.cpp

```diff
@@ -496,6 +496,21 @@ static const struct
     },
 };
 
+// Opposite Pitch values for reversed cars
```

Comment:
This table and the one at 2983 can be moved to VehiclePaint.cpp since it's the only user, imo.